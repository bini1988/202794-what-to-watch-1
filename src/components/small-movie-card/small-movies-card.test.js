import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card.jsx";

it(`SmallMovieCard correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <SmallMovieCard
          card={{
            id: `b15a1da5-8142-4d2a-b567-26599e333988`,
            title: `Movie Title`,
            genre: `Movie Genre`,
            img: `img/path`,
            trailer: `trailer/path`,
          }}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
