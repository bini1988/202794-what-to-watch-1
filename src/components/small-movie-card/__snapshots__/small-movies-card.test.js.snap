// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard correctly renders default markup 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <button
    className="small-movie-card__play-btn"
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Movie Title"
      height="175"
      src="img/path"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/film/b15a1da5-8142-4d2a-b567-26599e333988"
    >
      Movie Title
    </a>
  </h3>
</article>
`;
